docs/auth/oauth.md

```diff
@@ -99,42 +88,45 @@ Individual tokens will be accessible via their primary keys:
 `/api/<version>/me/oauth/tokens/<primary key of a token>/`. Here is a typical token:
 ```
 {
-    "id": 17,
-    "type": "access_token",
-    "url": "/api/v2/me/oauth/tokens/17/",
+    "id": 4,
+    "type": "o_auth2_access_token",
+    "url": "/api/v2/tokens/4/",
     "related": {
         "user": "/api/v2/users/1/",
-        "application": "/api/v2/me/oauth/applications/4/",
-        "activity_stream": "/api/v2/me/oauth/tokens/17/activity_stream/"
+        "application": "/api/v2/applications/1/",
+        "activity_stream": "/api/v2/tokens/4/activity_stream/"
     },
     "summary_fields": {
         "application": {
-            "id": 4,
-            "name": "admin's token",
-            "client_id": "D6SwhKbfp2LuUjkmiUpMMYFyNqhpv5PTVci7eXTT"
+            "id": 1,
+            "name": "Default application for root",
+            "client_id": "mcU5J5uGQcEQMgAZyr5JUnM3BqBJpgbgL9fLOVch"
         },
         "user": {
             "id": 1,
-            "username": "admin",
+            "username": "root",
             "first_name": "",
             "last_name": ""
         }
     },
-    "created": "2017-12-12T16:48:10.489550Z",
-    "modified": "2017-12-12T16:48:10.522189Z",
+    "created": "2018-02-23T14:39:32.618932Z",
+    "modified": "2018-02-23T14:39:32.643626Z",
+    "description": "App Token Test",
     "user": 1,
-    "token": "kqHqxfpHGRRBXLNCOXxT5Zt3tpJogn",
-    "refresh_token": "miZq3hqSugvYxhzdQYJIBDgIHxJPnT",
-    "application": 4,
-    "expires": "2017-12-13T02:48:10.488180Z",
+    "token": "*************",
+    "refresh_token": "**************",
+    "application": 1,
+    "expires": "2018-02-24T00:39:32.618279Z",
     "scope": "read"
-}
+},
 ```
-For an OAuth token, the only fully mutable field is `scope`. The `application` field is *immutable
+For an OAuth 2 token, the only fully mutable field is `scope`. The `application` field is *immutable
 on update*, and all other fields are totally immutable, and will be auto-populated during creation:
```

Comment:
> and all other fields are totally immutable

That's not right, `description` field can be modified, and I see no reason it shouldn't be modifiable. I would change the docs in this case.